.github/workflows/ci.yml

```diff
@@ -305,31 +305,106 @@ jobs:
           secret_vault_address: ${{ secrets.VAULT_ADDR }}
           secret_vault_roleId: ${{ secrets.VAULT_ROLE_ID }}
 
-  # description: Runs unit tests broken down by component and team. Tests are run by JUnit5 Suite and is invoked by the maven command
-  # test location: Operate
-  # owner: Core Features, Data Layer
-  consolidated-unit-tests:
-    if: needs.detect-changes.outputs.java-code-changes == 'true'
-    name: "[UT] ${{ matrix.component }} - ${{matrix.suiteName}}"
-    needs: [ detect-changes ]
-    timeout-minutes: 10
+  operate-backend-unit-tests:
+    if: needs.detect-changes.outputs.operate-backend-changes == 'true'
+    name: "[UT] Operate - ${{matrix.suiteName}}"
+    needs: [ detect-changes, setup-unit-tests ]
+    uses: ./.github/workflows/webapp-run-ut-reuseable.yml
     permissions: { }  # GITHUB_TOKEN unused in this job
-    runs-on: gcp-perf-core-8-default
+    secrets: inherit
     strategy:
       fail-fast: false
       matrix:
-        component: [ Operate ]
         suite: [ DataLayer, CoreFeatures ]
         include:
           - suite: DataLayer
             suiteName: Data Layer
           - suite: CoreFeatures
             suiteName: Core Features
+    with:
+      componentName: "Operate"
+      suite: ${{ matrix.suite }}
+
+  optimize-backend-unit-tests:
+    if: needs.detect-changes.outputs.optimize-backend-changes == 'true'
+    name: "[UT] Optimize - ${{matrix.suiteName}}"
+    needs: [ detect-changes, setup-unit-tests ]
```

Comment:
Where does this job `needs` the `setup-unit-tests` job actually? If it doesn't, we should avoid unnecessary dependencies.

Same for the other webapps unit test jobs.